sources/Core/Numerics/Matrix4x4.cs

```diff
@@ -128,5 +172,17 @@ public string ToString(string? format, IFormatProvider? formatProvider)
                 .Append('>')
                 .ToString();
         }
+
+        /// <summary>Creates a transposed version of this <see cref="Matrix4x4" />.</summary>
+        /// <returns>The transposed <see cref="Matrix4x4" />.</returns>
+        public static Matrix4x4 Transpose(Matrix4x4 m)
```

Comment:
```suggestion
        public static Matrix4x4 Transpose(Matrix4x4 value)
```

Rest of this file looks good.

Reply:
fixed